tests/flash_message_helper.go

```diff
@@ -0,0 +1,36 @@
+package tests
+
+import (
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/beego/beego/v2/server/web"
+)
+
+func GetFlash(cookies []*http.Cookie) *web.FlashData {
+	for _, cookie := range cookies {
+		if cookie.Name == "BEEGO_FLASH" {
+			return DecodeFlash(cookie)
+		}
+	}
+
+	return nil
+}
+
+func DecodeFlash(cookie *http.Cookie) *web.FlashData {
```

Comment:
Minor thing but since `DecodeFlash` is only used by `GetFlash`, it could be made "private" by not exporting it:

```suggestion
func decodeFlash(cookie *http.Cookie) *web.FlashData {
```